app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java

```diff
@@ -633,22 +633,32 @@ public void onPos(final SourceContent sourceContent, boolean b) {
                     Realm realm = Realm.getDefaultInstance();
                     WebLink link = realm.createObject(WebLink.class);
 
```

Comment:
Add statement to check if sourceContent is null. It will give null pointer exception if you do `sourceContent.getDescription().isEmpty()` and sourceContent is null.